src/components/TodoList.js

```diff
@@ -45,13 +45,21 @@ function TodoList() {
             return todo;
         })
         setTodos(updatedTodos);
+        console.log(todos)
     };
+    const deleteAll = id =>{
+        console.log(id)
+        const removeArr = [...todos].filter(todo => console.log(todo));
 
+        setTodos(removeArr);
+    }
+    
     return (
         <div className="todo-context">
             <h1>To-Do List</h1>
             <TodoForm onSubmit={addTodo} />
             <Todo todos={todos} completeTodo={completeTodo} removeTodo={removeTodo}  />
+            <button className="todo-button" onClick={deleteAll}>Clear All</button>
```

Comment:
The button text says "Clear All" and the function is named "deleteAll". Please make it consistent.

Reply:
Changes made